zinc/src/sbt-test/source-dependencies/constants/pending

```diff
@@ -1,5 +1,4 @@
 # Marked as pending, see https://github.com/sbt/sbt/issues/1543
-#
```

Comment:
Empty comments are not parsed correctly, so that's why they're removed. The parser error was not happening before because parser errors were recognized as legit scripted errors, even though they are not (!).
  
This means that a pending test could have given a false positive.